harness/determined/cli/config_policies.py

```diff
@@ -91,20 +124,21 @@ def delete_config_policies(args: argparse.Namespace) -> None:
                     cli.Arg(
                         "workload_type",
                         type=str,
-                        choices=["experiment", "ntsc"],
-                        help="the type (Experiment or NTSC ) of config policies",
+                        choices=["experiment", "tasks"],
+                        help="the type (Experiment or Tasks) of config policies",
                     ),
                     cli.Arg(
-                        "--workspace",
+                        "config_file",
                         type=str,
-                        required=True,  # change to false when adding --global
-                        help="config policies for the given workspace",
+                        help="path to the YAML or JSON file containing defined config policies "
+                        "(can be absolute or relative to the current directory)",
                     ),
                     cli.Arg(
-                        "--config-file",
+                        "-w",
+                        "--workspace-name",
```

Comment:
Why `--workspace-name` and not `--workspace`?

Reply:
I am cool with changing this back to `--workspace`, I guess I wanted to distinguish the param as a string rather than an int of the workspace ID, but any such misconception could easily be corrected with the `det config-policies <command> help`
@kkunapuli up to you! Should we go with `--workspace-name` or `--workspace`?

Reply:
hmm... well I suppose `--workspace-name` is not an unreasonable choice, since we take "project id" for `det e create`.

Ugh, but I wish we had standardized the CLI on either names or ids everywhere and this wasn't a concern.  But that isn't the fault of this pr, for sure.

Reply:
Looking at [this](https://github.com/determined-ai/determined/pull/9911#discussion_r1757159967) comment from when the original work was merged, it seems like the suggestion from model dev was also to use `--workspace-name` instead of `--workspace`, so I changed this back!

Reply:
Let's stick with `workspace-name`. There's already precedent with notebooks
```
% det notebook start -h

-w WORKSPACE_NAME, --workspace-name WORKSPACE_NAME
                                          workspace name (default: None)
```

Reply:
Gotcha! 
ok I'll change this back to `--workspace-name`.
Users can specify `--workspace` as a shorthand (or any substring shorter than "workspace-name") if they want to anyway!